rabbitmq/metadata.csv

```diff
@@ -78,223 +78,226 @@ rabbitmq.overview.messages.publish_in.count,gauge,,message,,Count of messages pu
+rabbitmq.erlang.vm.msacc.busy_wait_seconds.count,count,,second,,[OpenMetricsV2] Total time in seconds spent busy waiting. Without extra states this time is part of the 'other' state.,0,rabbitmq,busy wait seconds,
+rabbitmq.erlang.vm.msacc.check_io_seconds.count,count,,second,,[OpenMetricsV2] Total time in seconds spent checking for new I/O events.,0,rabbitmq,check io seconds,
+rabbitmq.erlang.vm.msacc.emulator_seconds.count,count,,second,,[OpenMetricsV2] Total time in seconds spent executing Erlang processes.,0,rabbitmq,emulator seconds,
+rabbitmq.erlang.vm.msacc.ets_seconds.count,count,,second,,[OpenMetricsV2] Total time in seconds spent executing ETS BIFs. Without extra states this time is part of the 'emulator' state.,0,rabbitmq,ets seconds,
+rabbitmq.erlang.vm.msacc.gc_full_seconds.count,count,,second,,[OpenMetricsV2] Total time in seconds spent doing fullsweep garbage collection. Without extra states this time is part of the 'gc' state.,0,rabbitmq,gc full seconds,
+rabbitmq.erlang.vm.msacc.gc_seconds.count,count,,second,,[OpenMetricsV2] Total time in seconds spent doing garbage collection. When extra states are enabled this is the time spent doing non-fullsweep garbage collections.,0,rabbitmq,gc seconds,
+rabbitmq.erlang.vm.msacc.nif_seconds.count,count,,second,,[OpenMetricsV2] Total time in seconds spent in NIFs. Without extra states this time is part of the 'emulator' state.,0,rabbitmq,nif seconds,
+rabbitmq.erlang.vm.msacc.other_seconds.count,count,,second,,[OpenMetricsV2] Total time in seconds spent doing unaccounted things.,0,rabbitmq,other seconds,
+rabbitmq.erlang.vm.msacc.port_seconds.count,count,,second,,[OpenMetricsV2] Total time in seconds spent executing ports.,0,rabbitmq,port seconds,
+rabbitmq.erlang.vm.msacc.send_seconds.count,count,,second,,[OpenMetricsV2] Total time in seconds spent sending messages (processes only). Without extra states this time is part of the 'emulator' state.,0,rabbitmq,send seconds,
+rabbitmq.erlang.vm.msacc.sleep_seconds.count,count,,second,,[OpenMetricsV2] Total time in seconds spent sleeping.,0,rabbitmq,sleep seconds,
+rabbitmq.erlang.vm.msacc.timers_seconds.count,count,,second,,[OpenMetricsV2] Total time in seconds spent managing timers. Without extra states this time is part of the 'other' state.,0,rabbitmq,timers seconds,
+rabbitmq.erlang.vm.port_count,gauge,,,,[OpenMetricsV2] The number of ports currently existing at the local node.,0,rabbitmq,port count,
+rabbitmq.erlang.vm.port_limit,gauge,,,,[OpenMetricsV2] The maximum number of simultaneously existing ports at the local node.,0,rabbitmq,port limit,
+rabbitmq.erlang.vm.process_count,gauge,,,,[OpenMetricsV2] The number of processes currently existing at the local node.,0,rabbitmq,process count,
+rabbitmq.erlang.vm.process_limit,gauge,,,,[OpenMetricsV2] The maximum number of simultaneously existing processes at the local node.,0,rabbitmq,process limit,
```

Comment:
```suggestion
rabbitmq.erlang.vm.process_count,gauge,,process,,[OpenMetricsV2] The number of processes currently existing at the local node.,0,rabbitmq,process count,
rabbitmq.erlang.vm.process_limit,gauge,,process,,[OpenMetricsV2] The maximum number of simultaneously existing processes at the local node.,0,rabbitmq,process limit,
```

Reply:
There might be other metrics that need the `process` unit that I might've missed